src/sidebar/services/test/view-filter-test.js

```diff
@@ -233,4 +240,38 @@ describe('sidebar/services/view-filter', () => {
 
     assert.deepEqual(result, [1]);
   });
+
+  describe('malformed target object', () => {
+    it('should not fail on annotations without a target object', () => {
```

Comment:
These two tests are a little different in that they dont test the result of the filter, rather they just make sure the code does not produce a js error.  If we remove either of those checks in view-test.js on line 103, these tests will fail. The fitler result itself does not change and is not important to the test.

If there is a better way to do this, please let me know